cgmanifest.json

```diff
@@ -2497,8 +2497,8 @@
         "type": "other",
         "other": {
           "name": "debootstrap",
-          "version": "1.0.128+nmu2",
-          "downloadUrl": "https://deb.debian.org/debian/pool/main/d/debootstrap/debootstrap_1.0.128+nmu2.tar.gz"
```

Comment:
it looks like the old .tar.gz isn't available from debian anymore, it seems to be replaced with <https://deb.debian.org/debian/pool/main/d/debootstrap/debootstrap_1.0.128+nmu2+deb12u1.tar.gz>